pkg/endpoints/cluster_test.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+		http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package endpoints
+
+import (
+	"encoding/json"
+	"net/http/httptest"
+	"testing"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestGetAllNamespaces(t *testing.T) {
```

Comment:
Can you add another short test please for a namespace that doesn't exist then we test that we return the 404?

Reply:
I don't think this is needed @a-roberts, the namesapce is added so there is a namespace to be returned, this is not adding testing getting a specific namespace ... rather getting all namespaces.

Reply:
Yep you're right, my mistake 😅 will resolve this convo...